src/CarShowroom/CarShowroom.js

```diff
@@ -1,84 +1,65 @@
-import React from "react";
+import { render } from "@testing-library/react";
+import React, {useState} from "react";
+import arrayFeatures from "../Data/arrayFeatures.json"
 
```

Comment:
if you dont use the render, let remove it. also separate the import:
```
import React, {useState} from "react";

import arrayFeatures from "../Data/arrayFeatures.json"

Reply:
ok.thanks. will do it